contrib/09-timezone

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
```

Comment:
shellcheck is legitimately fairly unhappy with this file. Please make sure it's shellcheck clean :-)